rikaicontent.js

```diff
@@ -748,6 +748,14 @@ var rcxContent = {
 		sel.removeAllRanges();
 		sel.addRange(range);
 		tdata.selText = sel.toString();
+
+		if(window.rikaichan.config.ttsEnabled == 'true') {
+			var text = sel.toString();
+			if(typeof text === 'string' && text.length > 0) {
+				console.log("Sending playTTS " + text);

```

Comment:
it turns out it's bad practice to log in content scripts, though useful for debugging.

Reply:
OK. I will comment that line.